app/templates/components/alert.html

```diff
@@ -1,27 +1,38 @@
 {%- from "components/alerts_icon.html" import alerts_icon -%}
 
-{% macro alert(alert, heading_level=2) %}
+{% macro alert(alert, heading_level=2, language='en') %}
   <div class="govuk-grid-row">
-    <div class="govuk-grid-column-two-thirds">
+    <div class="govuk-grid-column-full">
       <div class="alerts-icon__container alerts-icon__container--48">
         {{ alerts_icon(height=48, alert_active=alert.is_current) }}
         <h{{ heading_level }} class="alerts-alert__title govuk-heading-s govuk-!-margin-bottom-3">
-          {% if alert.is_public %}
-            <span class="govuk-visually-hidden">Emergency alert sent to </span>{{ alert.display_areas | formatted_list(before_each='', after_each='') }}
-          {% else %}
-            Mobile network operator test
-          {% endif %}
+            <span class="govuk-visually-hidden">Emergency alert sent to </span>{% if alert.display_areas %}{{ alert.display_areas | formatted_list(before_each='', after_each='') }}{% elif not alert.is_public %}Mobile network operator test{% endif %}
         </h{{ heading_level }}>
         {{ alert_body(alert) }}
         {% if alert.is_public %}
-          <a href="/alerts/{{ alert | get_url_for_alert }}" class="govuk-link govuk-body">
-            More information about this alert
-            <span class="govuk-visually-hidden">to {{ alert.display_areas | formatted_list(before_each='', after_each='') }}</span>
-          </a>
+          {% if language == 'cy' %}
+            <a href="/alerts/{{ alert | get_url_for_alert }}.cy" class="govuk-link govuk-body">
+              Rhagor o wybodaeth am y rhybudd hwn
+              <span class="govuk-visually-hidden">to {{ alert.display_areas | formatted_list(before_each='', after_each='') }}</span>
+            </a>
```

Comment:
Given we have embedded the entire welsh site in if statements within the same templates for now, it would be good to get some brief happy path view tests to ensure that things don't break and work as expected when Welsh is selected.

Reply:
You mean local link tests? We already have local link tests set up for .cy pages

Reply:
Actually I meant testing that we get welsh content when language is selected (not exhaustive, just to ensure that the text is in welsh).  Pretty sure we have some tests for the english version similar